tests/psoc6/dut/i2c_hard.py

```diff
@@ -0,0 +1,28 @@
+### I2C
+
+from machine import I2C
+import time
+import os
+
+# Allocate pin based on board
+machine = os.uname().machine
+if "CY8CPROTO-062-4343W" in machine:
+    scl_pin = "P6_0"
+    sda_pin = "P6_1"
+elif "CY8CPROTO-063-BLE" in machine:
+    scl_pin = "P6_4"
+    sda_pin = "P6_5"
+
+i2c = I2C(0, scl=scl_pin, sda=sda_pin, freq=400000)
+addr = i2c.scan()
+master_data = b"\x01\x00\x17"
+counter = 0
+
+while counter != 5:
```

Comment:
Yes! Though they do  work functionally right now. As soon as we enable slave this would look different. I did not want to remove them as I could manage to do the functional tests for both boards and add bare minimum to further build.